### PR TITLE
Add ReversingLabs playbooks to 6.1

### PR DESCRIPTION
**Pull request by MislavReversingLabs**
Add ReversingLabs playbooks created using the current schema.

### CODE REVIEW

**Comment by MislavReversingLabs** [comment]
@kelby-shelton Can you please assign someone to this pull request? Thank you.

**Comment by kelby-shelton** [comment]
@MislavReversingLabs After reviewing your playbooks, we believe these would be a good fit for our playbooks mapped to MITRE D3FEND. The two techniques I see here are Reputation Analysis, and Dynamic Analysis. We request that you combine the two reputation playbooks into a single input playbook that accepts either a File or Url and outputs a normalized output. It should also do error handling and only output with successful results. You can see an example of this here: https://github.com/phantomcyber/playbooks/blob/6.1/VirusTotal_v3_Identifier_Reputation_Analysis.png

For the detonation playbook, that can remain as-is but we would like you to add additional error handling and a normalized output as well. See https://github.com/phantomcyber/playbooks/blob/6.1/CrowdStrike_OAuth_API_Dynamic_Analysis.png and https://github.com/phantomcyber/playbooks/blob/6.1/Splunk_Attack_Analyzer_Dynamic_Analysis.png

Please let us know if you have any questions. I look forward to seeing the changes and having these two playbooks added to the community.

**Comment by MislavReversingLabs** [comment]
> @MislavReversingLabs After reviewing your playbooks, we believe these would be a good fit for our playbooks mapped to MITRE D3FEND. The two techniques I see here are Reputation Analysis, and Dynamic Analysis. We request that you combine the two reputation playbooks into a single input playbook that accepts either a File or Url and outputs a normalized output. It should also do error handling and only output with successful results. You can see an example of this here: https://github.com/phantomcyber/playbooks/blob/6.1/VirusTotal_v3_Identifier_Reputation_Analysis.png
> 
> For the detonation playbook, that can remain as-is but we would like you to add additional error handling and a normalized output as well. See https://github.com/phantomcyber/playbooks/blob/6.1/CrowdStrike_OAuth_API_Dynamic_Analysis.png and https://github.com/phantomcyber/playbooks/blob/6.1/Splunk_Attack_Analyzer_Dynamic_Analysis.png
> 
> Please let us know if you have any questions. I look forward to seeing the changes and having these two playbooks added to the community.

@kelby-shelton Thank you for the review.
What exactly do you mean when saying that our playbooks would be a good fit for your "playbooks mapped to MITRE D3FEND"?

Regarding merging our reputation playbooks into one playbook: we would like them to remain the way they are since we have other projects and plans oriented towards having these exact 5 playbooks published (2 of 5 remain for publishing after you implement looping into your new schema).

As for adding error handling into our playbooks, what would you recommend our playbook doing in case there is no reputation report? The actions from our apps used in these playbooks already implement error handling and raising in the Python SDK that we use. In your opinion, should we add a filtering block that just proceeds to the "End" block if the reputation request was not successful? Our two reputation playbooks have the "End" block set after doing the reputation request anyway.

**Comment by MislavReversingLabs** [comment]
@ljstella when can we expect a review ov this PR?

File: ReversingLabs_TitaniumCloud_File_Reputation.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        "python_version": "3",
+        "schema": "5.0.8",
+        "version": "5.5.0.108488"
+    },
+    "create_time": "2023-06-01T13:20:14.880779+00:00",
+    "draft_mode": false,
+    "labels": [
+        "*"
+    ],
+    "tags": []
```
Comment (kelby-shelton) [inline ReversingLabs_TitaniumCloud_File_Reputation.json]: "tags": [
    "Externally Authored Content"
 ]

File: ReversingLabs_TitaniumCloud_URL_Reputation.json [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        "python_version": "3",
+        "schema": "5.0.8",
+        "version": "5.5.0.108488"
+    },
+    "create_time": "2023-06-01T17:54:56.455210+00:00",
+    "draft_mode": false,
+    "labels": [
+        "*"
+    ],
+    "tags": []
```
Comment (kelby-shelton) [inline ReversingLabs_TitaniumCloud_URL_Reputation.json]: "tags": [
    "Externally Authored Content"
 ]

File: ReversingLabs_TitaniumScale_File_Analysis.json [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        "python_version": "3",
+        "schema": "5.0.8",
+        "version": "5.5.0.108488"
+    },
+    "create_time": "2023-06-01T15:17:52.635815+00:00",
+    "draft_mode": false,
+    "labels": [
+        "*"
+    ],
+    "tags": []
```
Comment (kelby-shelton) [inline ReversingLabs_TitaniumScale_File_Analysis.json]: "tags": [
    "Externally Authored Content"
 ]

File: ReversingLabs_TitaniumCloud_File_Reputation.json (hunk 1 above)
Comment (MislavReversingLabs) [reply ReversingLabs_TitaniumCloud_File_Reputation.json]: Done.

File: ReversingLabs_TitaniumCloud_URL_Reputation.json (hunk 2 above)
Comment (MislavReversingLabs) [reply ReversingLabs_TitaniumCloud_URL_Reputation.json]: Done.

File: ReversingLabs_TitaniumScale_File_Analysis.json (hunk 3 above)
Comment (MislavReversingLabs) [reply ReversingLabs_TitaniumScale_File_Analysis.json]: Done.

**Review by kelby-shelton — APPROVED** [review]

**Pull request merged**